app/src/main/java/com/snakyapps/khiardle/ui/Keyboard.kt

```diff
@@ -33,11 +33,7 @@ import androidx.compose.ui.unit.sp
 import com.snakyapps.khiardle.backend.models.EqualityStatus
 import com.snakyapps.khiardle.backend.models.KeyboardKeys
 import com.snakyapps.khiardle.backend.viewmodel.GameViewModel
-import com.snakyapps.khiardle.ui.theme.correctBackground
```

Comment:
We follow KtLint's standard rules which doesn't allow wildcard imports.

You can check it out [here](https://github.com/pinterest/ktlint).